utils/frame/remote-externalities/Cargo.toml

```diff
@@ -28,7 +28,7 @@ sp-core = { version = "3.0.0", path = "../../../primitives/core" }
 sp-runtime = { version = "3.0.0", path = "../../../primitives/runtime" }
 
 [dev-dependencies]
-tokio = { version = "1.6.0", features = ["macros", "rt"] }
+tokio = { version = "0.2", features = ["macros", "rt-threaded"] }
```

Comment:
Why the downgrade?

Reply:
Because the feature `tokio02` is enabled and must be spawned on tokio 0.2 to work.

The reason is that `sc-executor` is still using `tokio 0.2` and won't work otherwise or am missing something?

Reply:
Yeah I have seen the feature, but before this tokio version also worked?

Reply:
no, before we used `async-std` to spawn the background thread and switched for tokio instead.

Thus, must be spawned on either `tokio 0.2` or `tokio 1` to work or use `async-std` with tokio compat

Reply:
Okay, so before this was just a dead dependency. Fine

Reply:
right, sorry for the confusing but [I pointed out this to Kian when migrating to ws](https://github.com/paritytech/substrate/pull/8889/files#r637761815)